cleanup/cleanDockerImages/README.md

```diff
@@ -0,0 +1,59 @@
+Artifactory Clean Docker Images User Plugin
+===========================================
+
+This plugin is used to clean Docker repositories based on configured cleanup
+policies.
+
+Configuration
+-------------
+
+The `cleanDockerImages.properties` file has the following field:
+
+- `dockerRepos`: A list of Docker repositories to clean. If a repo is not in
+  this list, it will not be cleaned.
+
+For example:
+
+``` json
+dockerRepos = [
+// 		["repo_name", number_of_days_to_keep],
+//		if number_of_days_to_keep is empty, LABEL in Dockerfile with maxCount should be set.
```

Comment:
AnBut it is possible to define maxDays and maxCount, right?

Reply:
updated :)